pkg/k8s/slim/k8s/apis/meta/v1beta1/types.go

```diff
@@ -1,6 +1,18 @@
-// SPDX-License-Identifier: Apache-2.0
-// Copyright 2017 The Kubernetes Authors.
-// Copyright 2020-2021 Authors of Cilium
+/*
+Copyright 2017 The Kubernetes Authors.
+
```

Comment:
We lose the Cilium copyright notice on this file. Can you please confirm that this is intentional?

Reply:
ups, it's was not intentional.